lib/occ-helper.ts

```diff
@@ -38,7 +38,7 @@ export class OccHelper {
         edges.forEach((edge) => {
             const param1 = { current: 0 };
             const param2 = { current: 0 };
-            const crvHandle = this.occ.BRep_Tool.Curve_2(edge, param1.current, param2.current);
+            const crvHandle = this.occ.BRep_Tool.Curve_2(edge, param1 as any, param2 as any);
```

Comment:
heheh, yeah, so my first comment on your initial commit with some links to github discussions hopefully will explain this conundrum... I hate that "as any" as well here, but not sure how to make typescript happy with this... I think it's more the problem of Curve_2 constructor in opencascade.js than our codebase.

Reply:
Yes, the parameter type is just weird. Its basically number but wants {current: number}. Don't see why he didn't make a type. Anyway, doesn't matter. But I might suggest adding a note in the codebase to that discussion. And yeah I get that primitives are not passed by reference and it is internally setting the value in the code, a very C thing. You can pass a primitive in C(++) as &param or *param or if you're weird like in some of my previous code &*param.

Reply:
Those type definitions of opencascade are automatically generated, not sure how it works, but I can imagine that it simply parses c++ to come up with constructor types. I can imagine that emscripten does that or maybe some other tool, I'm really not sure about that part. Opencascade.js would need to manually intervene in this to fix this and hopefully at some point it will :)